src/pytest_benchmark/stats.py

```diff
@@ -207,6 +211,20 @@ def as_dict(self, include_data=True, flat=False, stats=True):
                 (k, funcname(v) if callable(v) else v) for k, v in self.options.items()
             )
         }
+        if self.cprofile_stats:
+            result["cprofile"] = []
```

Comment:
I would use an intermediate variable (eg `cprofile = result["cprofile"] = []`) for the loops bellow.